pilot/pkg/model/authorization.go

```diff
@@ -76,11 +76,14 @@ func (policy *AuthorizationPolicies) ListAuthorizationPolicies(selectionOpts Wor
 	if policy == nil {
 		return configs
 	}
+
 	rootNamespace := policy.RootNamespace
-	namespace := selectionOpts.Namespace
+	namespace := selectionOpts.WorkloadNamespace
```

Comment:
nit: should we just call this `workloadNamespace` and add `svcNamespace := selectionOpts.ServiceNamespace` too? Seems like it could be a little more consistent for readability.